### ISSUE TITLE
some questions about results table in the repo.

### ISSUE THREAD

**Issue by ignitemylife** [neuroethology/TREBA#1]
what's the difference between **w/ Task Programming** and **Domain-specific features** in the tabel **Results on Fly Dataset**?

As for the table **Results on Mouse Dataset**,  is there any information about the proportion of training data used in the experiments?

**Comment by jenjsun** [comment]
For the Fly dataset, the domain-specific features result corresponds to using the 100 dim per frame hand-designed features during classification. Adding TREBA w/ Task Programming means adding the 32 dim learned TREBA features w/ task programming as well (so each input dim is now 100 + 32 dim). 

For the mouse dataset, we used the full training set of tasks 1, 2, 3 in CalMS21 (and show the result on the test set). You can see the dataset paper (https://arxiv.org/pdf/2104.02710.pdf) for more information on the training set for each task.

**Comment by ignitemylife** [comment]
> For the Fly dataset, the domain-specific features result corresponds to using the 100 dim per frame hand-designed features during classification. Adding TREBA w/ Task Programming means adding the 32 dim learned TREBA features w/ task programming as well (so each input dim is now 100 + 32 dim).
> 
> For the mouse dataset, we used the full training set of tasks 1, 2, 3 in CalMS21 (and show the result on the test set). You can see the dataset paper (https://arxiv.org/pdf/2104.02710.pdf) for more information on the training set for each task.

Get it! Thanks for your reply. Best wishes

**Comment by jenjsun** [comment]
No prob, closing this issue then. :)
